app/test_main.py

```diff
@@ -1,3 +1,74 @@
+from typing import List, Any
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    # Basic test cases
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected",
+        [
+            (-1, 0, [0, 0]),
+            (0, -1, [0, 0]),
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
+        ],
+        ids=[
+            "Negative cat age",
+            "Negative dog age",
+            "Zero ages",
+            "Both animals are in young age",
+            "Both animals are in young age, just above the boundary",
+            "Boundary ages for young",
+            "Age just above the boundary",
+            "Boundary ages for adult",
+            "Age just above the boundary",
+            "Boundary ages for senior cat",
+        ]
+    )
+    def test_basic_cases(
+            self,
+            cat_age: Any,
```

Comment:
in this test cat_age and dog_age are always int type, expected is always a list of ints